sdk/tsconfig.json

```diff
@@ -1,21 +1,40 @@
 {
+  // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
```

Comment:
Should we make sure it is identical across all our repos (in another task)?
E.g., these are different:

**osx**
https://github.com/aragon/osx/blob/develop/packages/contracts/tsconfig.json
https://github.com/aragon/osx/blob/develop/packages/contracts-ethers/tsconfig.json
https://github.com/aragon/osx/blob/develop/packages/subgraph/tsconfig.json

**osx-plugin-template-hardhat**
https://github.com/aragon/osx-plugin-template-hardhat/blob/main/packages/contracts/tsconfig.json
https://github.com/aragon/osx-plugin-template-hardhat/blob/main/packages/subgraph/tsconfig.json